contracts/utils/ReentrancyGuard.sol

```diff
@@ -37,15 +50,13 @@ abstract contract ReentrancyGuard {
     uint256 private constant NOT_ENTERED = 1;
     uint256 private constant ENTERED = 2;
 
-    uint256 private _status;
```

Comment:
This is technically breaking, although the storage slot for this value doesn't really matter. Do we want to add a changelog entry for that?

Reply:
It is a storage change indeed, but I would argue its not breaking.